assets/src/apps/authoring/components/PropertyEditor/schemas/bank.ts

```diff
@@ -0,0 +1,129 @@
+import { UiSchema } from '@rjsf/core';
+import { IActivity } from 'apps/delivery/store/features/activities/slice';
+import {
+  SequenceEntry,
+  SequenceEntryChild,
+} from 'apps/delivery/store/features/groups/actions/sequence';
+import chroma from 'chroma-js';
+import { JSONSchema7 } from 'json-schema';
+import AccordionTemplate from '../custom/AccordionTemplate';
+import ColorPickerWidget from '../custom/ColorPickerWidget';
+import CustomFieldTemplate from '../custom/CustomFieldTemplate';
+
+const bankSchema: JSONSchema7 = {
+  type: 'object',
+  properties: {
+    Size: {
+      type: 'object',
+      title: 'Dimensions',
+      properties: {
+        width: { type: 'number' },
+        height: { type: 'number' },
+      },
+    },
+    palette: {
```

Comment:
I dont think you need all this stuff, just the items that go in the sequence for banks, just the 2 properties; the rest of the screen stuff doesn't exist in the sequence